tests/scripts/run-xunit-perf.py

```diff
@@ -0,0 +1,466 @@
+    (out, err) = proc.communicate()
+
+    if not proc.returncode == 0:
+        print_error(out, err, 'Failed to get information about the CLI tool.')
+
+    # Restore PerfHarness
+    perfHarnessPath = os.path.join(coreclrRepo, 'tests', 'src', 'Common', 'PerfHarness', 'PerfHarness.csproj')
+    run_args = [dotnet,
+            'restore',
+            perfHarnessPath]
+    log(" ".join(run_args))
+
+    proc = subprocess.Popen(run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=dotnetEnv)
+    (out, err) = proc.communicate()
+
+    if not proc.returncode == 0:
+        print_error(out, err, 'Failed to restore PerfHarness.csproj')
+
+    # Publish PerfHarness
+    run_args = [dotnet,
+            'publish',
+            perfHarnessPath,
+            '-c',
+            'Release',
+            '-o',
+            sandboxDir]
+    log(" ".join(run_args))
+
+    proc = subprocess.Popen(run_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=dotnetEnv)
+    (out, err) = proc.communicate()
+
+    if not proc.returncode == 0:
+        print_error(out, err, 'Failed to publish PerfHarness.csproj')
+
+def copytree(src, dst):
+    for item in os.listdir(src):
+        s = os.path.join(src, item)
+        d = os.path.join(dst, item)
+        if os.path.isdir(s):
+            shutil.copytree(s, d)
+        else:
+            shutil.copy2(s, d)
+
+def main(args):
+    coreclrPerf, arch, operatingSystem, configuration, jitName, optLevel, runType, outputDir, stabilityPrefix, isScenarioTest, benchviewPath, isPgoOptimized, benchviewGroup, hasWarmupRun, collectionFlags, isLibrary, uploadToBenchview, better, sliceNumber, sliceConfigFile = validate_args(args)
+
+    platform = sys.platform
+    python = 'py'
+    if platform == 'linux' or platform == 'linux2':
+        platform = 'Linux'
+        python = 'python3'
+    elif platform == 'darwin':
+        platform = 'OSX'
+        python = 'python3'
+    elif platform == 'win32':
+        platform = "Windows_NT"
+
+    executable = ['cmd.exe', '/c'] if platform == 'Windows_NT' else []
+
+    coreclrRepo = os.getcwd()
+    etwCollection = 'Off' if collectionFlags == 'stopwatch' else 'On'
+    sandboxDir = os.path.join(coreclrRepo, 'bin', 'sandbox')
+    sandboxOutputDir = outputDir if not outputDir is None else os.path.join(sandboxDir, 'Logs')
+
+    extension = '.exe' if platform == 'Windows_NT' else ''
+
+    my_env = os.environ
+    dotnetEnv = os.environ
+
+    dotnetEnv['DOTNET_MULTILEVEL_LOOKUP'] = '0'
+    dotnetEnv['UseSharedCompilation'] = 'false'
+
+    # Setup directories
+    log('Setting up directories')
+    coreclrOverlay = verify_core_overlay(coreclrRepo, platform, arch, configuration)
+    setup_sandbox(sandboxDir)
+
+    os.chdir(sandboxDir)
+
+    perfRunLog = set_perf_run_log(sandboxOutputDir)
+    build_perfharness(coreclrRepo, sandboxDir, extension, dotnetEnv)
+
+    my_env = setup_optimization_level(my_env, optLevel)
+
+    if not 'XUNIT_PERFORMANCE_MAX_ITERATION' in my_env:
```

Comment:
This was just added as a quality of life improvement? I assume that this should just stop local runs from defaulting to 1000 iters

Reply:
Yes. That was the purpose of this.